eknigge/GuideWalker
# Signout

# Removes the Welcome prompt on the map. Adds a log out button on the side drawer. When thelog out button is selected, it takes the user to a log out the confirmation page. After the user confirms, it takes the user to the log in screen. 

This PR addresses the following issues:

Closes #43  - Log Out - Add to side drawer

<img width="326" alt="Screen Shot 2021-11-19 at 4 39 26 PM" src="https://user-images.githubusercontent.com/47171828/142736463-ffd944d3-c8a6-4052-805b-8c6595e30d68.png">

<img width="338" alt="Screen Shot 2021-11-19 at 4 36 18 PM" src="https://user-images.githubusercontent.com/47171828/142736469-5200d796-658f-4701-b6cc-02eaff86aa9b.png">

<img width="332" alt="Screen Shot 2021-11-19 at 4 36 41 PM" src="https://user-images.githubusercontent.com/47171828/142736476-d7a01570-696a-4e5a-b2bd-877e5e25abe2.png">

<img width="321" alt="Screen Shot 2021-11-19 at 4 57 37 PM" src="https://user-images.githubusercontent.com/47171828/142736481-8c1aa382-6ad9-494a-a87c-157731a4e62e.png">

<img width="325" alt="Screen Shot 2021-11-20 at 9 58 34 AM" src="https://user-images.githubusercontent.com/47171828/142736493-3e909654-dd25-4226-b423-4275ec317628.png">
## How to test

1. Run the app and log in.
2. Notice that the Welcome prompt is no longer visible. 
2. Select the navigation button on the top left.
3. A side drawer will appear, select Log Out on the side drawer.
4. Confirm the Log Out.

## Checklist

- [x] Did you request a review from someone?
